src/app/asset-readings/assets/assets.component.html

```diff
@@ -48,16 +48,20 @@ <h5 class="title is-5"> Assets
   <div class="card">
     <header>
       <div id="card-title" class="card-content">
-        <h5 class="title is-5">Asset Readings</h5>
+        <h5 class="title is-5">Asset Readings
+          <button (click)="getAsset(); setAssetCode(selectedAsset)" class="button is-small">
```

Comment:
is this a chained call? what if selectedAsset is not inside latest getAsset result? will it show `Select` or get blank?

Reply:
yes It is kind of chained call and  if `selectedAsset ` is not there then it will show `Select`.